CorsixTH/Lua/diseases/jellyitis.lua

```diff
@@ -30,6 +30,7 @@ disease.cure_price = 1000
 disease.emergency_sound = "emerg014.wav"
 disease.emergency_number = 12
 disease.contagious = false
+disease.effect = AnimationEffect.Jelly
```

Comment:
Nothing to change, but careful we're not creeping into the jellyitis pr

Reply:
This was an intentional choice. This PR adds the animation patent_effect and persists it to save games, so I decided it would be nice to get both effects saved correctly. Otherwise in https://github.com/CorsixTH/CorsixTH/issues/1946 I'd have to add more complex savegame restoration logic to only restore radiation in 158 and jelly in 159. Having this here means that the game stores that the patient should use the jelly effect (and it is removed when treated), but it is not visualized yet.

Reply:
Okay happy then :)